pkg/db/interface.go

```diff
@@ -341,6 +343,201 @@ func (d *dbConn) DeleteStudy(id string) error {
 	return err
 }
 
+func (d *dbConn) CreateNAS(in *api.StudyConfig) (string, error) {
```

Comment:
The level is different between CreateNAS and CreateStudy. Please change them to an equal level e.g. CreateNASStudy, CreateHPStudy.